### ISSUE TITLE
[REVIEW] Krish Gupta

### ISSUE THREAD

**Issue by krishdevdb** [EddieHubCommunity/support#1995]
**Please ensure that you don't have any personal details in your Portfolio that you don't want to share publicly.**

Portfolio link: @krishdevdb 

You can also join the Discord community [here](http://discord.eddiehub.org)

Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)

**Comment by schmelto** [comment]
Nice profile @krishdevdb 

some thoughts
* remove the "Opensource Projects" Section -> duplicate with pinned repos
* think the stats are overall really big maybe you could align them with less space

**Comment by krishdevdb** [comment]
@schmelto I respect your feedback. I didn't knew you were so fast, I thought that I will make a new profile completely from scratch until someone sees this. Today I deleted my old profile to make changes thus, kinda your feedback went unused sorry 😢 But first I thought to work upon github's new features, rebrand my self (use my original image [Inspired by Eddie's livestream which I had to watch 3 times to understand the original image thing])... So I am closing this as that might take some time thanks! I really appreciate your feedback.
